lib/country_picker_dropdown.dart

```diff
@@ -11,6 +11,7 @@ class CountryPickerDropdown extends StatefulWidget {
     this.itemBuilder,
     this.initialValue,
     this.onValuePicked,
+    this.isExpandEnabled
```

Comment:
I think calling it isExpanded should be enough

Reply:
Updated the flag name.